cmd/root.go

```diff
@@ -71,16 +71,24 @@ Find more information at https://github.com/kubeapps/kubeapps.`,
 }
 
 func init() {
-	RootCmd.PersistentFlags().CountP("verbose", "v", "Increase verbosity.")
+	RootCmd.PersistentFlags().StringP("verbose", "v", "warning", fmt.Sprint("Set log level: debug, info, warning, error.\n"+
```

Comment:
the usual value for the default is `info`, isn't it?

Reply:
yes default to `info` makes sense. The `warning` was there for a while